examples/linear_model/plot_logistic_multinomial.py

```diff
@@ -0,0 +1,68 @@
+"""
+====================================================
+Plot multinomial and One-vs-Rest Logistic Regression
+====================================================
+
+Plot decision surface of multinomial and One-vs-Rest Logistic Regression.
+The hyperplanes corresponding to the three One-vs-Rest (OVR) classifiers
+are represented by the dashed lines.
```

Comment:
I fear the multinomial example is not very visual with the iris dataset:
ovr:
![ovr](https://cloud.githubusercontent.com/assets/11065596/11559188/362931be-99b7-11e5-9dd5-b214ea480090.png)
multinomial:
![multinomial](https://cloud.githubusercontent.com/assets/11065596/11559189/362b4710-99b7-11e5-8082-3579b7211fb9.png)